.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: node_js
+node_js:
+- '6.9'
+- '7.7'
+cache:
+  yarn: true
+  directories:
+    - node_modules
+    - packages/neutrino-middleware-chunk/node_modules
```

Comment:
Should this list also include these entries?
* `packages/neutrino-middleware-banner/node_modules`
* `packages/neutrino/node_modules`